apps/app/src/styles/tailwind.css

```
@@ -75,6 +75,10 @@
 	.border-aside {
 		border-image: linear-gradient(90deg, transparent, rgba(0, 0, 0, 0.08) 50%, transparent) 100;
 	}
+
+	.forceLockScroll {
```

Czemu ta klasa jest zapisana w inny sposób niż wszystkie pozostałe?
```suggestion
	.force-lock-scroll {
```

---

Poza tym, ta klasa nie jest potrzebna, w tailwindzie jest `overflow-hidden`, a żeby dodać `!important` wystarczy ją poprzedzić wykrzyknikiem: `!overflow-hidden`

---

Zastosowana konwencja nazewnictwa klasy CSS spowodowana jest chyba tym, że kiedyś bawiłem się styles components 🤮 btw poprawie to.

Co do możliwości dodania deklaracji `!important` do kasy z tailwinds przez poprzedzenie nazwy klasy wykrzyknikiem, to muszę sprawdzić, czy nie wymaga to jakiejś dodatkowej konfiguracji, bo chyba coś takiego było, i czy nie wpływa to jakkolwiek negatywnie na sposób stylowania stosowany do tej pory w projekcie.